shared/chat/conversation/input-area/normal/platform-input.native.js

```diff
@@ -335,7 +335,7 @@ const styles = styleSheetCreate({
   },
   container: {
     ...globalStyles.flexBoxRow,
-    alignItems: 'flex-end',
+    alignItems: 'center',
```

Comment:
are you sure you want this. does this look on when there's multiple lines?

Reply:
`actionButton` has its own `alignSelf` so we're good.